docs/topics/permissions.rst

```diff
@@ -77,9 +80,27 @@ Using the *Pages global permissions* model you can give a set of permissions to
 
 .. note:: You always **must** set the sites managed py the global permissions, even if you only have one site.
 
+.. _structure_mode_permissions:
+
+***************
+Mode permission
+***************
+
+.. versionchanged:: 3.1
+
+django CMS uses **Structure** and **Content** modes for different type of content editing;
+while the former allows full control over the plugins layout, positioning and to add new
+plugins to the page, the latter only allow editing existing plugins.
+
+Since version 3.1 the specific permission "**Can use Structure mode**" exists to allow restricting
+the access to structure mode; this allows to define different level of permissions on the
+same content.
 
```

Comment:
Consider: "From version 3.1 the specific permission "**Can use Structure mode**" exists to permit access to Structure Mode. This allows defining a different level of permissions on the same content."